interface/main/calendar/find_appt_popup.php

```diff
@@ -176,7 +176,7 @@ function doOneDay($catid, $udate, $starttime, $duration, $prefcatid): void
         array_push($sqlBindArray, $providerid, $eid, $sdate, $edate, $sdate, $edate);
 
     // phyaura whimmel facility filtering
-    if ($_REQUEST['facility'] ?? '' > 0) {
+    if ($_REQUEST['facility'] ?? 0 > 0) {
```

Comment:
@openemr/admins Er, I was about to blame rector for this, but it didn't do anything wrong. `>` is higher precedence than `??`, so this comparison is almost certainly incorrect.

brb, I'm going to go see if this is a whole class of bugs in openemr.